pytest.ini

```diff
@@ -1,4 +1,4 @@
 [pytest]
-addopts = --cov=adsdata
-testpaths = tests
-python_files = *.py
+addopts = --cov=adsdata --cov-report=term-missing
+testpaths = tests/
```

Comment:
coverage decreased by 9% - which is weird - i'm guessing one of these two lines are responsible